src/components/Tutorials/subComps/RemoveStepModal.jsx

```diff
@@ -6,88 +6,90 @@ import { useDispatch } from "react-redux";
 
-	useEffect(() => {
-		setVisible(viewModal);
-	}, [viewModal]);
+  useEffect(() => {
+    setVisible(viewModal);
+  }, [viewModal]);
 
-	const handleOnOk = () => {
-		<Snackbar
-			anchorOrigin={{
-				vertical: "bottom",
-				horizontal: "left",
-			}}
-			open={true}
-			autoHideDuration={6000}
-			message="Updating...."
-		/>;
-		if (step_length > 1) {
-			removeStep(
-				owner,
-				tutorial_id,
-				step_id,
-				currentStep
-			)(firebase, firestore, dispatch).then(() => {
-				setVisible(false);
-				<Snackbar
-					anchorOrigin={{
-						vertical: "bottom",
-						horizontal: "left",
-					}}
-					open={true}
-					autoHideDuration={6000}
-					message="removed...."
-				/>;
-			});
-		}
-	};
-	const handleOnCancel = () => setVisible(false);
+  const handleOnOk = () => {
+    <Snackbar
+      anchorOrigin={{
+        vertical: "bottom",
+        horizontal: "left"
+      }}
+      open={true}
+      autoHideDuration={6000}
+      message="Updating...."
+    />;
+    if (step_length > 1) {
+      removeStep(
+        owner,
+        tutorial_id,
+        step_id,
+        currentStep
+      )(firebase, firestore, dispatch).then(() => {
+        setVisible(false);
+        <Snackbar
+          anchorOrigin={{
+            vertical: "bottom",
+            horizontal: "left"
+          }}
+          open={true}
+          autoHideDuration={6000}
+          message="removed...."
+        />;
+      });
+    }
+  };
+  const handleOnCancel = () => setVisible(false);
 
-	return (
-		<Modal
-			open={visible}
-			onClose={handleOnCancel}
-			aria-labelledby="simple-modal-title"
-			aria-describedby="simple-modal-description"
-			style={{
-				border: "2px solid #000",
-				background: "whitesmoke",
-				boxShadow: "2rem gray",
-				display: "flex",
-				alignItems: "center",
-				justifyContent: "center",
-				height: "10rem",
-				width: "20rem",
-				position: "absolute",
-				top: "40%",
-				left: "40%",
-			}}>
-			<div>
-				<Typography>This action is can not be undone!</Typography>
-				<form onSubmit={handleOnOk}>
-					<Button key="back" onClick={handleOnCancel}>
-						<Typography>Cancel</Typography>
-					</Button>
-					<Button key="submit" type="primary" htmlType="submit">
-						<Typography> Remove</Typography>
-					</Button>
-				</form>
-			</div>
-		</Modal>
-	);
+  return (
+    <Modal
+      open={visible}
+      onClose={handleOnCancel}
+      aria-labelledby="simple-modal-title"
+      aria-describedby="simple-modal-description"
+      style={{
+        border: "2px solid #000",
+        background: "whitesmoke",
+        boxShadow: "2rem gray",
+        display: "flex",
+        alignItems: "center",
+        justifyContent: "center",
+        height: "10rem",
+        width: "20rem",
+        position: "absolute",
+        top: "40%",
+        left: "40%"
+      }}
+    >
+      <div>
+        <Typography>This action is can not be undone!</Typography>
+        <Stack direction="row" spacing={1}>
```

Comment:
Great Work!!!
May I know why we are using `Stack` here instead of `form`

Reply:
Hello, i used stack only to keep items in a row with a spacing of 1.

Reply:
actually we just have to prevent the page from reloading when a step is removed.
This gives us the time to execute remove step function completely and remove the step.
Can you use `form` and make sure the page doesnt reload when the remove step is called.

Reply:
actually we just have to prevent the page from reloading when a step is removed.
This gives us the time to execute remove step function completely and remove the step.
Can you use `form` and make sure the page doesnt reload when the remove step is called.

Reply:
@shivareddy6 Sure will do that, thank you

Reply:
Hi @shivareddy6 i have pushed the changes and provided fix using the ```form``` tag.
thanks